**Задание 1:** Случайная непрерывная величина A имеет равномерное распределение на промежутке (200, 800].
Найдите ее среднее значение и дисперсию.

In [1]:
import numpy as np
import pandas as pd
import matplotlib.pyplot as plt

In [3]:
a = 200
b = 800
M_x = (a + b) / 2
M_x

500.0

In [4]:
D_x = ((b-a)**2)/12
D_x

30000.0

**Задание 2:** О случайной непрерывной равномерно распределенной величине B известно, что ее дисперсия равна 0.2.
Можно ли найти правую границу величины B и ее среднее значение зная, что левая граница равна 0.5?
Если да, найдите ее.

Запишем уравнение: ((b-0.5)**2)/12=0.2

Приведем к виду: b**2 - b - 2.15 = 0

Решим уравнение:

In [10]:
np.roots([1, -1, -2.15])

array([ 2.04919334, -1.04919334])

Правая граница случайной величины B = 2.04919334, так как правая граница > левой, а -1.04919334 < 0.5

Тогда среднее значение:

In [15]:
M_x = (2.04919334 + 0.5) / 2
M_x

1.27459667